flex.go

```diff
@@ -101,6 +101,12 @@ func (f *Flex) RemoveItem(p Primitive) *Flex {
 	return f
 }
 
+// Clear removes all items from the flexbox
```

Comment:
Can you change this to the following, please?

> Clear removes all items from the container.

(Notice the period at the end.)

It may seem like a small and irrelevant change but I kept  the entire package consistent in this regard and would like to keep it this way. See also the [contributing guidelines](https://github.com/rivo/tview/blob/master/CONTRIBUTING.md):

> Start all sentences upper-case, as is common in English, and _end them with a period._